### ISSUE TITLE
Identify and track App Suggestions

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/mozilla-vpn-client#6853]
Engineering can fine tune this task, but at a high level we need a system that:

* Checks the currently installed apps on the user’s device against a set of known apps that we intend to recommend excluding from VPN protection.
* Can run on-demand to get a ‘refreshed version’ upon various triggers.
* Is able to track which apps are newly found vs present in previous runs. This is because we intend to notify users (only once) about app suggestions.
* Exposes the model for suggested app exclusions as a part of the current app exclusion model, or a new one

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/VPN-4742)
┆Reporter: Santiago Andrigo

**Comment by data-sync-user** [comment]
➤ Matt Cleinman commented:

Closing this in favor of ticket breakdown that matches the tech spec
